TodoList/bot/clients/todoClient.ts

```diff
@@ -97,8 +97,20 @@ export async function getTodoItemData(taskId: number, token: string, isViewer: b
             currentStatus: resp[0].currentStatus,
             taskContent: resp[0].taskContent,
             creator: creator,
+            sharedwith: "",
             viewers: []
         }
+        
+        if (resp.length == 1 && resp[0].viewerId == null) {
+            return todoItem;
+        }
+
+        todoItem.sharedwith = resp[0].viewerId;
+        for (let i: number = 1; i < resp.length; ++i) {
+            todoItem.sharedwith = todoItem.sharedwith + "," + resp[i].viewerId;
+        }
+        todoItem.sharedwith = todoItem.sharedwith.toLowerCase();
```

Comment:
as a todoClient, it should only take single responsibility: query db and return result with predefined schema. UI related logic should be decoupled from todoClient so we don't need to ajust it's logic. The logic may be moved to `botService.ts`.